apps/frontend/editor/src/components/form/ArrayInput/README.md

```
@@ -0,0 +1,139 @@
+# ArrayInput System
+
+## Why?
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)